glossarpy/GlossEntry.py

```diff
@@ -1,67 +1,8 @@
 import datetime
```

Comment:
`build/` should probably be in your `.gitignore` file and not in your main code base.